content/community/organization/governance.html

```diff
@@ -39,7 +39,7 @@ <h2>Haiku Support Association (HSA)</h2>
 <p>The <a href="http://haiku-support-association.org/index-eng.html">Haiku Support Association</a> is a German based organisation of users and developers of Haiku. Their objective is to promote the development and further adoption of Haiku, which is done by focusing their efforts on organizing the regularly held BeGeistert event and its associated CodeSprint.</p>
 
 <h2>HaikuPorts</h2>
-<p><a href="https://github.com/haikuports/haikuports">HaikuPorts</a> is a project consisting of of Haiku users, developers and contributors that tries to get applications and software libraries that are available on other operating systems working on Haiku.</p>
+<p><a href="https://github.com/haikuports/haikuports">HaikuPorts</a> is a project consisting of Haiku users, developers, and contributors that tries to get applications and software libraries that are available on other operating systems working on Haiku.</p>
```

Comment:
Is the oxford comma correct here?

To me it now reads like the "that tries to get applications..." part applies only to "contributors", and not to "HaikuPorts".

I would have done this instead:

```suggestion
<p><a href="https://github.com/haikuports/haikuports">HaikuPorts</a> is a project consisting of Haiku users, developers and contributors, that tries to get applications and software libraries that are available on other operating systems working on Haiku.</p>
```

Maybe a native english speaker can comment on this?